### PR TITLE
Exception fixes

### PR DESCRIPTION
**Pull request by vext01**
Bugs found in OpenBSD VM.

See commit messages for details.

OK?

### CODE REVIEW

File: krun.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def main(parser):
 
     try:
         inner_main(mailer, on_first_invocation, config, args)
-    except Exception:
+    except Exception as e:
```
Comment (snim2) [inline krun.py]: Oops. Can we not have one letter variables though, please?

**Comment by snim2** [comment]
One minor comment, but otherwise looks good

**Comment by vext01** [comment]
Please wait on merging.

**Comment by vext01** [comment]
Here we are. I though I had screwed something up, but I was running the wrong krun checkout >.<

OK to squash?

**Comment by snim2** [comment]
Yep, good to squash!

**Comment by vext01** [comment]
splat.

**Comment by snim2** [comment]
OK, will merge when Travis is done.

**Pull request merged**